ethereum/chain.py

```diff
@@ -263,21 +264,21 @@ def add_block(self, block):
                         break
                     b = self.get_parent(b)
                 replace_from = b.header.number
-                for i in xrange(replace_from, 2**63 - 1):
+                for i in range(replace_from, 2**63 - 1):
```

Comment:
Using `itertools.count` makes a lot of sense. This will also remove the dependency on `future` which I added because I didn't realize I could just change it to `itertools.count`.